src/views/Room.vue

```diff
@@ -95,7 +98,7 @@ export default {
       inputAddUrlText: "",
 
       showJoinFailOverlay: false,
-      joinFailReason: ""
+      joinFailReason: "",
```

Comment:
For future reference, avoid making changes like this that don't pertain to the feature/bug you are working on. It helps to avoid merge conflicts and keep commits clean.